mirgecom/integrators/explicit_rk.py

```diff
@@ -28,11 +28,18 @@
 """
 
 
-def rk4_step(state, t, dt, rhs):
+def rk4_step(state, t, dt, rhs, reference_state=None):
```

Comment:
It seems that this might all be avoidable by `partial`'ing the `reference_state` into `rhs` as a keyword argument. (For all the steppers.)